scripts/build_scenario1.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from jinja2 import Environment, FileSystemLoader
+
+# Définir le répertoire contenant le template
+template_dir = '/home/chausser/test/'
```

Comment:
the template should be included within the repo (so that the script can be run everywhere)